### DISCUSSION TITLE
You should be able to delete publish profiles from within the Composer UI

### DISCUSSION THREAD

**Discussion by irwinwilliams** [microsoft/BotFramework-Composer#5775] [General]
![image](https://user-images.githubusercontent.com/1428703/107720682-350d3000-6cb1-11eb-874a-518520a0d81e.png)

You can edit them, but to delete, you have to go into AppSettings.json. My suggestion is to provide a delete button.

**Comment by a-b-r-o-w-n** [comment]
We are already tracking this feature request: #5642
